scripts/orchestra_methods.py

```diff
@@ -0,0 +1,319 @@
+#! /usr/bin/env python
```

Comment:
Please add module-level documentation at the top of this file.  

You can automatically transclude such file overviews as output for the `--help` command-line argument (more on the at bottom of review) by using [`RawTextHelpFormatter`](https://docs.python.org/2/library/argparse.html#argparse.RawTextHelpFormatter) and `description=__doc__` in your ArgParse constructor.

Reply:
Added help